api/src/main/resources/application.yml

```diff
@@ -32,4 +32,9 @@ logging:
     console: "%d{HH:mm:ss.SSS} [%thread] [%X{session.id}][%X{request.id}] %-5level %logger{36} - %msg%n"
   level:
     ROOT: ${vcap.services.variable-service.credentials.dwp-api-app-root-loglevel:info}
-    uk.gov.dhsc: ${vcap.services.variable-service.credentials.dwp-api-app-loglevel:debug}
\ No newline at end of file
+    uk.gov.dhsc: ${vcap.services.variable-service.credentials.dwp-api-app-loglevel:debug}
+
+dwp:
+  eligibility-check-in-weeks: 4
```

Comment:
As discussed, we shouldn't be defining this in this repository - one for a future pr